packages/app/src/cli/models/extensions/specifications/app_config_app_access.ts

```diff
@@ -47,6 +47,10 @@ const appAccessSpec = createConfigExtensionSpecification({
   schema: AppAccessSchema,
   transformConfig: AppAccessTransformConfig,
   getDevSessionUpdateMessages: async (config) => {
+    if (config.access_scopes?.use_legacy_install_flow === true) {
+      return [`Using legacy install flow - access scopes are not auto-granted`]
+    }
```

Comment:
I opened this PR thinking this was going to be a big change, so happy to see it was this easy :)